imagetagger/imagetagger/settings.py.docker

```diff
@@ -0,0 +1,172 @@
+"""
+Django settings for imagetagger project.
+
+Generated by 'django-admin startproject' using Django 1.10.3.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/1.10/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/1.10/ref/settings/
+"""
+
+import os
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/1.10/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '9x)d3h#dczqi^4zupz6d_s(jmtsm1cdjgkt4fb=w)%4$!^cyn6'
```

Comment:
You should not make it that easy to forget changing the secret key. Furthermore, I have not seen any hint that it should be changed in your installation document!